src/pages/AddItemPage.jsx

```diff
@@ -0,0 +1,11 @@
+import AddItemForm from "../components/AddItemForm";
+
+function App() {
```

Comment:
```suggestion
function AddItemPage() {
```
컴포넌트 이름이 그대로인것같아요! 아래 export하는 부분도요!